src/library/searchqueryparser.h

```diff
@@ -27,6 +32,8 @@ class SearchQueryParser {
     QString getTextArgument(QString argument,
                             QStringList* tokens) const;
 
+    TrackCollection* m_pTrackCollection;
+
     QSqlDatabase m_database;
```

Comment:
m_database is redundant and should instead be obtained from m_pTrackCollection if needed